src/services/inplayer.account.service.ts

```diff
@@ -25,14 +39,36 @@ export const login: Login = async ({ config, email, password }) => {
     });
 
     return {
-      auth: processInPlayerAuth(data),
-      user: processInplayerAccount(data.account),
+      auth: processAuth(data),
+      user: processAccount(data.account),
     };
   } catch {
     throw new Error('Failed to authenticate user.');
   }
 };
 
+export const register: Register = async ({ config, email, password }) => {
+  try {
+    const { data } = await InPlayer.Account.signUpV2({
+      email,
+      password,
+      passwordConfirmation: password,
+      fullName: 'New User',
```

Comment:
Can this be blank instead?

Reply:
No actually.. We have a backend validation for at least 3 letters i think..

Reply:
Oh, that makes it difficult. Maybe we could use the email address then at least?  So it's not something that seems like a bug?  Longer term we can either look at changing the backend or changing the web app flow to not actually make the register call until the second step of the form where name is entered.

Reply:
Yes, we can do that.